apps/torus-wallet/src/app/_components/review-transaction-dialog.tsx

```
@@ -57,27 +58,25 @@ export const ReviewTransactionDialog = forwardRef<
-      toast({
-        title: "Error",
-        description: err.message,
-        variant: "destructive",
-      });
-      onError?.(err);
-    } finally {
-      setIsSubmitting(false);
+      return;
     }
+
+    setIsSubmitting(false);
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)